Update coexistence-chat-calls-presence.md (#7755)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/coexistence-chat-calls-presence.md

Both the in-tenant and federated behaviors described above are available, with the following limitations:

- External attendees whose tenants reside in a different GoLocal deployment or geography won't see IM chat while in a "federated" meeting

> - Federation and interop between Multitenant O365 and Office 365 running by 21vianet is supported with limited scenarios.

Reviewer: ```suggestion
- Federation and interop between Multitenant O365 and Office 365 operated by 21Vianet is supported in limited scenarios.
```

In particular, if a recipient you previously had a persistent interop conversation thread with was upgraded to Teams, that thread will no longer reflect accurate presence and will no longer be routable. You should start a new thread.

> ### Federation and interop with 21vianet

Reviewer: ```suggestion
### Federation and interop with Office 365 operated by 21Vianet
```

> Federation and interop between Multitenant O365 and 21vianet O365 is supported when the Multitenant O365 users are in TeamsOnly mode. In such a scenario, SFBO users in 21vianet will be able to communicate with TeamsOnly users in Multitenant O365 through chats and calls. The following table shows the supported scenarios in this configuration:

Reviewer: ```suggestion
Federation and interop between Multitenant O365 and Office 365 operated by 21Vianet are supported when the Multitenant O365 users are in TeamsOnly mode. In such a scenario, Skype for Business Online users in Office 365 operated by 21Vianet will be able to communicate with TeamsOnly users in Multitenant O365 through chats and calls. The following table shows the supported scenarios in this configuration:
```

 
|Scenario|Origin|Recipient|Supported?|
|---|---|---|---|
|Presence|Teams <br> Skype for Business <br> | Skype for Business <br> Teams|Yes<br>Yes|

> |Chat|Teams <br> Skype for Business <br> | Skype for Business <br> Teams|Yes (1:1 Only)<br>Yes(1:1 Only)|
> |Audio Calls|Teams <br> Skype for Business <br> | Skype for Business <br> Teams|Yes (1:1 Only)<br>Yes (1:1 Only)|
> |Video Calls|Teams <br> Skype for Business <br> | Skype for Business <br> Teams|Yes (1:1 Only)<br>Yes (1:1 Only)|
> |Screen Sharing|Teams <br> Skype for Business <br> | Skype for Business <br> Teams |Yes (through a promoted Teams meeting)<br>Yes (through a promoted SfB meeting)|

Reviewer: ```suggestion
|Chat|Teams <br> Skype for Business <br> | Skype for Business <br> Teams|Yes (1:1 only)<br>Yes(1:1 only)|
|Audio Calls|Teams <br> Skype for Business <br> | Skype for Business <br> Teams|Yes (1:1 only)<br>Yes (1:1 only)|
|Video Calls|Teams <br> Skype for Business <br> | Skype for Business <br> Teams|Yes (1:1 only)<br>Yes (1:1 only)|
|Screen Sharing|Teams <br> Skype for Business <br> | Skype for Business <br> Teams |Yes (through a promoted Teams meeting)<br>Yes (through a promoted Skype for Business meeting)|
```
